README.md

```diff
@@ -1,22 +1,34 @@
 # Reaction Storefront Next.js Starter Kit
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Freactioncommerce%2Freaction-next-starterkit.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Freactioncommerce%2Freaction-next-starterkit?ref=badge_shield)
 
-
 _**Note:** This project is a work in progress and should not be used in production at this time._
 
-Reference Storefront application for [Reaction Commerce](https://reactioncommerce.com/).
+Reference headless ecommerce storefront starter kit application for [Reaction Commerce](https://reactioncommerce.com/), featuring [React](https://reactjs.org/), [Next.js](https://nextjs.org/) and [Apollo](https://www.apollographql.com/docs/react/) to connect to the [Reaction GraphQL API](https://github.com/reactioncommerce/reaction/), available as of Reaction v2.0.0.
+
+## Features
+
+- Headless ecommerce starter kit with [Next.js](https://nextjs.org/), [React](https://reactjs.org/), [MobX](https://mobx.js.org/getting-started.html), [GraphQL](https://graphql.org/), [Apollo](https://www.apollographql.com/docs/react/)
+- Server-side rendering
+- Payments with [Stripe](https://stripe.com/)
+- Analytics with [Segment](https://segment.com/)
+- Reusable, customizable, themeable ecommerce React components from the new [Reaction Component Library](https://www.npmjs.com/package/@reactioncommerce/components) with [Styled Components](https://www.styled-components.com/)
+- Fully-configured test suite: Jest snapshot testing, Mocha integration testing
+- Written in ES6, configured with ES6
+- Containerized with Docker
 
 ## Getting Started
```

Comment:
Lets have people start using this storefront by using the reaction platform, which will simplify things and remove most of this manual setup process. Plus, that is why the reaction platform project exists.